deployment/postgres-init-db/sql/tables/actions/action_run.sql

```diff
@@ -76,10 +79,29 @@ begin
     select pg_notify('action_run_inserted', json_strip_nulls(row_to_json(payload))::text)
     from payload
   );
-  return null;
 end$$;
 
 create trigger notify_action_run_inserted
   after insert on actions.action_run
   for each row
 execute function actions.notify_action_run_inserted();
+
+create function actions.notify_action_run_cancel_requested()
+  returns trigger
+  security definer
+  language plpgsql as $$
+begin
+  perform pg_notify('action_run_cancel_requested', json_build_object(
+      'action_run_id', NEW.id
+  )::text);
+end$$;
```

Comment:
Missing `return null;` at the last line. The function says it `returns trigger`, you need to return _something_. (`null` is picked because this is a function for an _after_ trigger, though you could reasonably return `new`.